src/native/eventpipe/ds-ipc.c

```diff
@@ -390,18 +420,18 @@ ds_ipc_stream_factory_get_next_available_stream (ds_ipc_error_callback_func call
 							saw_error = true;
 						_ds_current_port = port;
 					}
-					DS_LOG_INFO_2 ("ds_ipc_stream_factory_get_next_available_stream - SIG :: Poll attempt: %d, connection %d signalled.\n", poll_attempts, connection_id);
+					DS_LOG_DEBUG_2 ("ds_ipc_stream_factory_get_next_available_stream - SIG :: Poll attempt: %d, connection %d signalled.", poll_attempts, connection_id);
```

Comment:
You only changed this one to `DS_LOG_DEBUG*`.  Did you mean to change all of them?

Reply:
Was thinking about that first, but since the others are special conditions that might be interesting to see on lower log level, I just switched the normal case to higher since that is what happens frequently when running Diagnostic Server.